src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -20,12 +20,14 @@ export async function pollUntil(fn: () => (boolean | Promise<boolean>), timeoutM
     const resolvedPollInterval = pollInterval ? pollInterval : 50;
 
     let timeWaited = 0;
-    let fnEval = fn();
-    if (fnEval instanceof Promise) {
-        fnEval = await fnEval;
-    }
+    let fnEval;
```

Comment:
Fun fact: anywhere we were using `pollUntil` before this was only actually checking the value the first time, then quietly hitting the timeout because most places have `suppressError == true`. This should make the tests faster, and in my local testing did seem to do so.